cmd/kubectl-cro/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"os"
+
+	"github.com/kro-run/kro/cmd/kubectl-cro/render"
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "kubectl-cro",
```

Comment:
How about just `kro` for a CLI name? also curious why cro and not kro?

Reply:
`kubectl-kro` was the intention - the `c` is a typo :)

I basically just took that from the issue, which used the CLI as a `kubectl` extension. But I have no strong opinions (in fact I might also prefer to just call it `kro`, and have it live side by side with `kubectl` rather than "within" it).

Reply:
Sweet! I'm not set on any specific way either. I've been using `kubectl kro` (k kro) for a while now and it feels a bit clunky. I was thinking what if we just call it "kro" and let people decide for themselves if they want to use it as a kubectl plugin or standalone?